src/Hl7.Fhir.ElementModel/MaskingNode.cs

```diff
@@ -65,10 +65,12 @@ public MaskingNode(ITypedElement source, MaskingNodeSettings settings = null)
                 ies.ExceptionHandler = (o, a) => ExceptionHandler.NotifyOrThrow(o, a);
         }
 
-        private MaskingNode(MaskingNode parent, ITypedElement source)
+        private MaskingNode(MaskingNode parent, ITypedElement source, bool? includeAll = null)
         {
             Source = source;
-            _settings = parent._settings;
+            _settings = parent._settings.Clone();
```

Comment:
No, we cannot do a clone - for performance reasons.  Every single child in every single instance represented by a MaskingNode will now get a copy of the settings, which will cause unnecessary GC pressure.  So, we should store this override in a local variable instead.